h/accounts/__init__.py

```diff
@@ -96,3 +100,8 @@ def includeme(config):
     config.include('.schemas')
     config.include('.subscribers')
     config.include('.views')
+
+    secret = config.registry.settings['secret_key']
+    derived = derive_key(secret, b'h.accounts')
+    serializer = URLSafeTimedSerializer(derived)
+    config.registry.password_reset_serializer = serializer
```

Comment:
Alice is just copying the existing pattern for adding these serializers. I think this is fine.